### DISCUSSION TITLE
Trying to install code server on termux showing error

### DISCUSSION THREAD

**Discussion by gotesgan** [coder/code-server#3358] [Q&A] [answered]
Hii guys trying to install code-server by using global yarn v1.22.10
But showing unexpected error 
![IMG-20210512-WA0000](https://user-images.githubusercontent.com/84043270/117875893-52f22d00-b2c0-11eb-9ec2-febec26546c8.jpg)

**Comment by jsjoeio** [comment, marked as answer]
This sounds like a network error. There's an interesting discussion [here](https://github.com/npm/npm/issues/6820) with some tips.

Are you using a proxy or do you have a firewall on your device?

**Reply by gotesgan** [reply]
No i am using the moblie data and I don't have any network issue still getting this problem I checked twise everything

**Reply by jsjoeio** [reply]
Inside Termux, can you run `ping registry.npmjs.org` and post the results?

**Reply by gotesgan** [reply]
Thanks actually it was the network problem I have solved the problem
